upf/upf_classify.c

```diff
@@ -278,6 +278,26 @@ upf_acl_classify_forward (vlib_main_t * vm, u32 teid, flow_entry_t * flow,
 	      *pdr_idx = acl->pdr_idx;
 
 	    far = pfcp_get_far_by_id (active, pdr->far_id);
+	    flow->ipfix_context_index = is_ip4 ?
+	      far->ipfix_context_index_ip4 : far->ipfix_context_index_ip6;
+	    /*
+	     * If IPFIX policy is not set in the FAR, use the value
+	     * from the session
+	     */
+	    if (flow->ipfix_context_index == (u32) ~ 0)
```

Comment:
If you unconditionally use the sessions IPFIX settings, how can a FAR then disable the inclusion of its flow in the export?
If should IMHO be possible to specify a empty IPFIX policy at the FAR level and use that to override any session level or default IPFIX export rules.

Reply:
Fixed, now it's possible to specify an empty string as IPFIX policy in the FAR to override the NWI setting.